README.md

```diff
@@ -50,6 +50,10 @@ LayoutKit is a fast view layout library for iOS, macOS, and tvOS.
 Layout is done by setting UIView's frame property directly. This implementation comes directly from the LayoutKit benchmark.
 [Manual layout benchmark's source code](https://github.com/layoutBox/LayoutFrameworkBenchmark/blob/master/LayoutFrameworkBenchmark/Benchmarks/ManualLayout/FeedItemManualView.swift)
 
+* [**NKFrameLayoutKit**](https://github.com/kennic/NKFrameLayoutKit)  
+NKFrameLayoutKit is a fast and easy to use layout library  
+[NKFrameLayoutKit benchmark's source code](https://github.com/kennic/LayoutFrameworkBenchmark/blob/master/LayoutFrameworkBenchmark/Benchmarks/NKFrameLayoutKit/NKFrameLayoutKitView.swift)
```

Comment:
You should point immediately to "https://github.com/layoutBox/LayoutFrameworkBenchmark/blob/master/..."